app/views/admin/vote_users/_adjustments.html.erb

```diff
@@ -2,18 +2,31 @@
     <%= fa_icon('plus') %> <%= t('adjustment.new') %>
 <% end %>
 
-<%= grid(@attend_grid) do |grid|
-  grid.column name: t('log.type'), attribute: 'presence', filter: false do |a|
-    in_out_table(a.presence)
-  end
-  grid.column name: model_name(Agenda), assoc: :agenda, attribute: 'title', filter: false do |a|
-    a.agenda
-  end
-  grid.column name: t('edit') do |a|
-    link_to t('edit'), edit_admin_adjustment_path(a)
-  end
-  grid.column name: t('remove') do |a|
-    link_to t('remove'), admin_adjustment_path(a), method: :delete, data:
-            {confirm: t('vote.confirm_remove') + '?'}
-  end
-end -%>
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th><%= t('log.type') %></th>
+      <th><%= model_name(Agenda) %></th>
+      <th><%= t('edit') %></th>
+      <th><%= t('remove') %></th>
+    </tr>
+  </thead>
+  <tbody id="adjustments-sortable">
+    <% @adjustments.each do |adjustment| %>
+      <tr data-item-id=<%= "#{adjustment.id}" %> class="item">
+        <td>
+          <%= in_out_table(adjustment.presence) %>
+        </td>
+        <td>
+          <%= adjustment.agenda %>
+        </td>
+        <td>
+          <%= link_to t('edit'), edit_admin_adjustment_path(adjustment) %>
+        </td>
+        <td class='adjustments-deleterow'>
+          <div class='glyphicon glyphicon-remove'></div>
```

Comment:
I would prefer `<%= fa_icon('times') %>`
I get some assets errors from the glyphicons